### PR TITLE
Add "Just fight!" map by Astatine-MT

### PR DESCRIPTION
**Pull request by Astatine-MT**
Adds a small map that makes fighting skill required to dominate.

### CODE REVIEW

**Comment by ClobberXD** [comment]
Screenshots please! :)

**Comment by Astatine-MT** [comment]
Screenshot from 2019-06-22 11-27-16
Screenshot from 2019-06-22 10-54-48

**Comment by Astatine-MT** [comment]
IMPORTANT: Bug found, trying to fix.

File: fight.conf [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+team.2.pos = (42,-4.5,-42)
+team.1.pos = (-42,-4.5,42)
+team.1.color = red
+team.1 = red
+author = Astatine
+rotation = x
+h = 11
+team.2 = blue
+name = Just fight!
+initial_stuff = default:sword_steel, default:pick_steel, default:stonebrick 40, medkits:medkit 3, ctf_bandages:bandage 5, default:torch 4
```
Comment (xenonca) [inline fight.conf]:
```suggestion
initial_stuff = default:sword_steel,default:pick_steel,default:stonebrick 40,medkits:medkit 3,ctf_bandages:bandage 5,default:torch 4
```

**Comment by xenonca** [comment]
For some reason there's always a part of the wall which doesn't disappear after the match starts:
jf

**Comment by Astatine-MT** [comment]
Yeah. Also, the flags weirdly disappear.

**Comment by Astatine-MT** [comment]
Screenshot from 2019-06-22 11-43-55
Yeah, not sure why.

**Pull request closed without merging**